backend/lib/server_code/api/roomRoutes.js

```diff
@@ -1,11 +1,16 @@
 const express = require("express");
-const { socket } = require("./socket.js");
+const { createNewSocket } = require("./socket.js");
+
+const socketsCollection = {};
 
 let roomRouter = express.Router();
 
 // Mockups
 let roomCreated = false;
 let roomPlayers = [];
+const roomIDCreator = () => {
```

Comment:
when you create a mock, it should make sense. 
like, roomIdCreator should get as a parameter the room ID

Reply:
I ment generattor, fixed af259f1